wiki/MapData.md

```diff
@@ -438,6 +448,12 @@ Additionally, giving it a description will cause the wormhole to create a link o
 
 **Since v. 0.10.0**: Create a `wormhole` node and then assign it to the planet using `wormhole <name>` (see below).
 
+```html
+"display name" <name>
+```
+
+Added in **v. 0.10.11**, this name is displayed to the player. If not defined, it defaults to the true name of the planet. Multiple planets can share the same display name.
```

Comment:
```suggestion
Since **v. 0.10.11**: Specifies the name displayed to the player. If not defined, it defaults to the true name of the planet. Multiple planets can share the same display name.
```